projects/ancestor/ancestor.py

```diff
@@ -1,3 +1,33 @@
+from graph import Graph
+from util import Stack
+
 
 def earliest_ancestor(ancestors, starting_node):
-    pass
\ No newline at end of file
+    graph = Graph()
+    # make a graph inserting all of the ancestors
+    for parent, child in ancestors:
+        graph.add_vertex(parent)
+        graph.add_vertex(child)
+    
+    for parent, child in ancestors:
```

Comment:
Excellent adding the edges, we could also add this to the loop above, but this works!